Using a library that consumes funcadelic throws an error in Jest

I started a [new repo](https://github.com/microstates/react/tree/2bf4e9283cc4353b858af93a82d0c624d409ff62) that consumes microstates library. 

When I run tests in jest, I get the following error `TypeError: Cannot read property 'Symbol(@std/esm:module._extensions[".mjs"])' of undefined`

```
➜  react git:(master) ✗ npm run test

> @microstates/react@0.1.0 test /Users/tarasmankovski/Repositories/microstates/react
> jest

 FAIL  tests/connect.test.jsx
  ● Test suite failed to run

    TypeError: Cannot read property 'Symbol(@std/esm:module._extensions[".mjs"])' of undefined

      at Object.<anonymous> (node_modules/funcadelic/nodule.js:2:11)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.569s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @microstates/react@0.1.0 test: `jest`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @microstates/react@0.1.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tarasmankovski/.npm/_logs/2017-12-11T15_29_50_792Z-debug.log
```

I attached the log [2017-12-11T15_29_50_792Z-debug.log](https://github.com/cowboyd/funcadelic.js/files/1548348/2017-12-11T15_29_50_792Z-debug.log)


To reproduce, 
1. clone https://github.com/microstates/react/tree/2bf4e9283cc4353b858af93a82d0c624d409ff62
2. `npm install`
3. `npm test`

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "test": "mocha tests/index.js"
   },
   "dependencies": {
-    "@std/esm": "^0.14.0",
+    "@std/esm": "^0.18.0",
     "lodash": "^4.17.4",
     "object.getownpropertydescriptors": "2.0.3"
   },

